src/Illuminate/Database/Connectors/PostgresConnector.php

```diff
@@ -81,33 +81,41 @@ protected function configureTimezone($connection, array $config)
     }
 
     /**
-     * Set the schema on the connection.
+     * Set the search_path on the connection.
      *
      * @param  \PDO  $connection
      * @param  array  $config
      * @return void
      */
-    protected function configureSchema($connection, $config)
+    protected function configureSearchPath($connection, $config)
     {
-        if (isset($config['schema'])) {
-            $schema = $this->formatSchema($config['schema']);
+        if (isset($config['searchpath']) || isset($config['schema'])) {
+            $searchPath = $this->formatSearchPath($config['searchpath'] ?? $config['schema'] ?? '"public"');
```

Comment:
This line looks a bit weird with the chain of ??

Reply:
I'm sorry can you elaborate on "looks weird" ?

Reply:
It looks like a new protected method would work better than chaining the null coalesce operator

Reply:
Can you elaborate on "better"? Hehe... seriously, though, this seems like the "simplest" solution to me. If there is some worthwhile advantage to be gained by adding a new method, then feel free to propose the change.

Reply:
Not to sound smug but, "work better" ? It would work no differently then what is here ? If you are referring to readability, then the change is merely an option. This implementation does not violate any coding principles or practices. Please be more clear with your comments and maybe contribute with an example.

Thanks in advance

@jmarcher

Reply:
And this is precisely why I left it alone: https://github.com/laravel/framework/pull/22425